src/server.rs

```diff
@@ -1,23 +1,51 @@
+#![allow(deprecated)]
 use std::collections::HashMap;
+use std::fmt;
 use std::rc::Rc;
 use std::sync::{Arc, Mutex};
 
-use flux::semantic::nodes::FunctionParameter;
+use flux::analyze;
+use flux::ast::walk::walk_rc;
+use flux::ast::walk::Node as AstNode;
+use flux::ast::{Expression, Package, PropertyKey, SourceLocation};
+use flux::parser::parse_string;
+use flux::semantic::nodes::Expression as SemanticExpression;
+use flux::semantic::nodes::{CallExpr, FunctionParameter};
+use flux::semantic::types::{MonoType, Record};
 use flux::semantic::walk;
-use log::{debug, error, info, warn};
-use lspower::jsonrpc::Result;
-use lspower::lsp;
-use lspower::LanguageServer;
+use flux::semantic::walk::Node as SemanticNode;
+use flux::semantic::walk::Visitor as SemanticVisitor;
+use flux::{imports, prelude};
 
-use crate::convert;
+use crate::convert::node_to_location;
```

Comment:
Why this change? I would prefer not to pollute the namespace here too much, if we can help it.

Reply:
I think this is the result of me moving a bunch of functions in and then backing a lot of it back out. I'll revert it.